spec/controllers/carriers_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe CarriersController, type: :controller do
+  describe "GET index" do
+    let(:query) do
+      {
+        origin: {
+          :country=>"US",
+          :state=>"CA",
+          :city=>"Beverly Hills",
+          :zip=>"90210"
+        },
+        destination: {
+          :country=>"US",
+          :state=>"WA",
+          :city=>"Seattle",
+          :zip=>"98103"},
```

Comment:
This is a nitpick, but it's good practice to be consistent with your styling. So I would suggest putting the `},` on a new line to match the other hashes.